db/fixtures/server_roles.csv

```diff
@@ -1,5 +1,6 @@
 name,description,max_concurrent,external_failover,role_scope
 automate,Automation Engine,0,false,region
+cockpit_ws,Cockpit,1,false,region
```

Comment:
If it's only ssh, that might be ok.  Typically, we put things that need to talk to a provider in the same zone as the provider.